### ISSUE TITLE
Installed for 4.1 - Set up errors

### ISSUE THREAD

**Issue by seanbydesign** [frumbert/wp2moodle--wordpress-#8]
1. renamed zip
2. uploaded
3. installed 
4. Activated 
Saw this: 

http://i.imgur.com/SWTbOJh.png

Clicked the wp2Moodle button to start setup: got this:

http://i.imgur.com/YntBr8K.png

Anyone have issues with 4.1? My site auto updated on my dev server while I wasn't looking, and I had to start from scratch it was broken so bad. I have a stable build on 4.1 and this was the final tie in for my client. Getting moodle and wordpress talking. 

Thanks.

**Comment by frumbert** [comment]
The wordpress 2 moodle demo site is running Wordpress 4.1 and also has auto-updates enabled. Perhaps there's another issue. Look at `phpinfo()` for your site and check that your PHP has the mcrypt extension enabled. Also, you might want to enable debugging messages in your wordpress (having it say "Error" isn't that helpful) - you have to modify your `wp-config.php` file and add the line `define('WP_DEBUG', true);`

**Comment by seanbydesign** [comment]
Installed mycrypt, deleted and re-installed plugin, restarted Apache. Didn't work. Debug set to true, only getting "error", then finally tried restarting my VPS. The icon isn't even loading up, but the plugin is appearing on the menu.

**Comment by frumbert** [comment]
hmmm. are the permissions all ok?

**Comment by seanbydesign** [comment]
Yeah, permissions were all okay. Just cant get it working. I hope its not a theme conflict. Second. Does this work for instillation of Moodle that are on other servers? I notice all the demo shows it as a WP subdomain and a moodle subdomain.

**Comment by frumbert** [comment]
Yeah, it works between any two servers. In its original installation, Wordpress was running on Windows, and Moodle on a Debian VPS. They don't share a database or anything - the information about users is passed across in an encrypted string. Can you bring up another out-of-the-box Wordpress on the same server with the plugin on it?

**Comment by hallshouse** [comment]
Hi:
I'm running WP 4.2.2.
I'm trying to use your version 1.5 plugin from GitHub.

The plugin is not working.
Here is what I did:
1. Extracted and uploaded wp2moodle folder to my plugins folder.
2. Activated WP2Moodle plugin.
3. Tried to launch the wp2moodle menu item in Wp-Admin menu.
...and got a 404 error on the wp-admin/wp2moodle/wp2m_settings_page.php page.
...WP2Moodle did not create the wp-admin/wp2moodle folder so I created the folder myself and copied the wp2m_settings_page.php file into it as well as the wp2m.js file (to be safe).
4. Set the wp-admin/wp2moodle folder and file permissions to 777.
5. Set the plugins/wp2moodle folder and file permissions to 777.
6. Refreshed my FF screen and found no change. I'm still getting the 500 page error.

Any advice would be great.

**Comment by actXc** [comment]
I do not undrestand your workflow.
in 1) you uploaded the folder
in 2)  you where able to activate
in 3) there is no folder anymore ???
I had no trouble following the instructions on https://github.com/frumbert/wp2moodle--wordpress-
No need to change permissions.

Note, you must to rename the zip to be just 'wp2moodle.zip' before you upload the plugin to wordpress. If the zip extracts to a sub-folder, it won't work!

1. Upload this to your wordpress (should end up being called /wp-content/plugins/wp2moodle/)
2. Activate the plugin
3. Click wp2moodle on the wordpress menu
4. Set your moodle url (e.g. http://your-site.com/moodle/)
5. Set the shared secret. This is a salt that is used to encrypt data that is sent to Moodle. Using a guid (http://newguid.com) is a good idea. It must match the shared secret that you use on the Moodle plugin. (https://github.com/frumbert/wp2moodle-moodle)
6. Other instructions are available on the settings page.

if you need professional support on your installation, I can offer remote support.

**Comment by hallshouse** [comment]
Thanks for the quick reply actXc:

For 1) Yes, I did not upload the zip file, I extracted the zip file on my local PC and renamed the folder it extracted into to "wp2moodle" and FTP'ed it into my wp-content/plugins folder.
To be as clear as I can be: The WP2Moodle plugin is in my wp-content/plugins and the folder name is wp2moodle and in that folder I have the icon.png, wp2m.js, wp2m.php and wp2m_settings_page.php files along with the Readme.md file.

For 2) Yes, I activated the wp2moodle plugin without issue.

For 3) The wp-content/plugins/wp2moodle folder and contents do exist. What did not get created during the installation/activation process was the wp-admin/wp2moodle folder and its contents.

FYI: I just successfully installed and configured it. using the FTP upload of the wp2moodle folder from my local drive.
During this process, I was making a recording for you to see exactly what I'm doing. I chose to use FF for this installation. The result was that the upload of the wp2moodle.zip file failed. I then went to my FTP and uploaded the folder, then activated the plugin.
This time, when I clicked the wp2moodle admin menu link it worked.

I then removed and reinstalled in using IE11 again and it all failed again. Installing this plugin using FF always succeeds while installing using IE11 always fails. I do not have this issue with other plugins in IE11.

Best Regards,
Dennis Hall
